kibble/render/translations_api_renderer.go

```diff
@@ -0,0 +1,67 @@
+//    Copyright 2018 SHIFT72
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package render
+
+import (
+	"encoding/json"
+	"fmt"
+	"kibble/models"
+	"os"
+	"path/filepath"
+)
+
+// Ensures language files exist for languges specified via Translations API
+func ensureLanguageFiles(site *models.Site, sourcePath string) error {
```

Comment:
- Is the golang convention not to have PascalCase names for functions used outside of the package?
- Is this the best name for this function? Maybe "WriteLanguageFiles" would be more apt?

Reply:
`ensureLanguageFiles` is only used within the render.go package so I'm not sure it needs to be made public?
'WriteLanguageFiles' was its original name @stajs wanted to make it clearer that this was called only if the API was being used, but that check has been moved outside of this functions so I see no harm in changing it back 😆